File: com.getshop.pullserverjar/src/com/getshop/pullserver/Store.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.getshop.pullserver;

import java.util.Date;
import java.util.List;

/**
 *
 * @author ktonder
 */
public class Store extends DataCommon {
    public String webAddress;
    public String webAddressPrimary;
    public List<String> additionalDomainNames;
    public boolean registeredDomain=false;
    public boolean readIntroduction=false;
    public boolean isExtendedMode=true;
    public boolean isVIS = false; //Very important shop.
    public boolean isDeepFreezed = false;
    public String deepFreezePassword = "";
    public boolean premium = false;
    public boolean mobileApp = false;
    
    public Date expiryDate;
    
    /**
     * This specifies if this store is setup as a template
     * If it is a template it can be cloned into another store :D
     */
    public boolean isTemplate = false;

}

File: html5apps/terminal/events/core/storemanager/data/Store.php
<?php
class core_storemanager_data_Store extends core_common_DataCommon  {
	/** @var String */
	public $webAddress;

	/** @var String */
	public $webAddressPrimary;

	/** @var String */
	public $identifier;

	/** @var String */
	public $additionalDomainNames;

	/** @var String */
	public $registeredDomain;

	/** @var String */
	public $readIntroduction;

	/** @var String */
	public $isExtendedMode;

	/** @var core_storemanager_data_StoreConfiguration */
	public $configuration;

	/** @var String */
	public $isVIS;

	/** @var String */
	public $isDeepFreezed;

	/** @var String */
	public $deepFreezePassword;

	/** @var String */
	public $premium;

	/** @var String */
	public $mobileApp;

	/** @var String */
	public $favicon;

	/** @var String */
	public $masterStoreId;

	/** @var String */
	public $acceptedSlaveIds;

	/** @var String */
	public $defaultMultilevelName;

	/** @var String */
	public $expiryDate;

	/** @var String */
	public $isTemplate;

	/** @var core_usermanager_data_User */
	public $registrationUser;

	/** @var String */
	public $country;

	/** @var String */
	public $timeZone;

	/** @var String */
	public $acceptedGDPR;

	/** @var String */
	public $acceptedGDPRDate;

	/** @var String */
	public $acceptedByUser;

}
?>